test/unit_tests/braket/default_simulator/openqasm/test_interpreter.py

```diff
@@ -1193,12 +1193,23 @@ def test_bit_operators():
 def test_input(in_int):
```

Comment:
Do we also have to the test the error thrown if input is not the integer/bit string literal but being tried to assign to one, or is it tested somewhere else?

Reply:
We test elsewhere if an input variable is missing. We also test elsewhere if an instantiated variable is the incorrect type.